core/dbt/docs/README.rst

```diff
@@ -0,0 +1,5 @@
+dbt-core
+=======
+
+**dbt-core** An ELT tool for managing your SQL transformations and data models.
```

Comment:
Just for understanding, is this file used to generate some header for the generated doc?

Reply:
I'm actually not sure if this is necessary, this isn't part of what is generated. I'm attempting to remove it and test if everything still works as intended, but I'm having some issues with mypy stubs. I'll check once I resolve those.

Reply:
Removed!